### ISSUE TITLE
Align caption and titles in maps

### ISSUE THREAD

**Issue by caldwellst** [OCHA-DAP/hdx-signals#149]
Align captions and titles in maps. The distance between them looks particularly bad for countries that are longer rather than wider, like Chile and Vietnam.

![image](https://github.com/OCHA-DAP/hdx-signals/assets/8086547/b76fbada-abc3-4ac4-92f5-f320bcdcc0ab)

Can be fixed in the same PR as #148.

**Comment by zackarno** [comment]
also just noting that in this example the break figures on the bubble look odd? 21, 22, 22?

**Comment by caldwellst** [comment]
Yeah good catch. Did not expect this behaviour.

```r
 scales::pretty_breaks(n=3)(c(1,2,2))
#> [1] 1.0 1.5 2.0
```

So it's rounding 2 numbers into 3. Need to add a line to `map_points.R` that checks the number of unique values and uses that as the number of breaks if lower than 3.

```r
min(length(unique(df[[val_col]])), 3)
```

**Comment by hannahker** [comment]
@caldwellst @zackarno is this addressed by #151?

**Comment by caldwellst** [comment]
yes!
